diff --git a/dist/runtime/Icon.vue b/dist/runtime/Icon.vue
index e68a5d8c5ed7b18f2314a5c76576c71f8dcd2f3c..ee2cb7b054e1f99513d5acfbc154df6d00d2c3fd 100644
--- a/dist/runtime/Icon.vue
+++ b/dist/runtime/Icon.vue
@@ -39,7 +39,7 @@ watch(() => appConfig.nuxtIcon?.iconifyApiOptions, () => {
     new URL(appConfig.nuxtIcon.iconifyApiOptions.url)
   } catch (e) {
     console.warn('Nuxt Icon: Invalid custom Iconify API URL')
-    return
+    // return
   }
 
   // don't override the default public api if publicApiFallback is enabled. See more: https://iconify.design/docs/api/providers.html
